### PR TITLE
fix(tooling): upgraded webpack packages and included polyfills

### PR DESCRIPTION
**Pull request by Kesari3008**
Putting permanent fix for demo:build issue. Upgraded webpack and included polyfills

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +32,8 @@
   ],
   "dependencies": {
     "@webex/component-adapter-interfaces": "^1.30.5",
-    "@webex/sdk-component-adapter": "1.112.6",
     "@webex/components": "^1.274.2",
+    "@webex/sdk-component-adapter": "1.112.6",
```
Comment (sreenara) [inline package.json:36]: Any reason for this change?

File: webpack.config.js [hunk 2]
```diff
@@ -14,12 +14,20 @@ module.exports = function(env, argv) {
...
           }
         : undefined, // Otherwise the CleanWebpackPlugin will wipe our build during devserver
-    node: {
-      fs: 'empty', // Webex SDK `fs` dependency does not exist in browser.
-    },
+    node: {},
     devtool: argv.mode === 'production' ? 'source-map' : 'inline-source-map',
     resolve: {
       extensions: ['.js', '.jsx'],
+      fallback: {
```
Comment (sreenara) [inline webpack.config.js:21]: Could we have comments around what these changes are for?

File: webpack.config.js [hunk 3]
```diff
@@ -63,13 +71,18 @@ module.exports = function(env, argv) {
...
+            static: {
+              directory: path.resolve(__dirname, './demo'),
+            },
             open: true,
-            overlay: true,
             hot: true,
             port: 9000,
-            stats: 'errors-warnings',
-            https: true,
+            client: {
```
Comment (sreenara) [inline webpack.config.js:80]: here as well

File: webpack.config.js [hunk 4]
```diff
@@ -82,7 +95,11 @@ module.exports = function(env, argv) {
       new webpack.HotModuleReplacementPlugin(),
       new webpack.DefinePlugin({
         __appVersion__: JSON.stringify(version)
-      })
+      }),
+      new webpack.ProvidePlugin({
```
Comment (sreenara) [inline webpack.config.js:99]: and here

**Review by sreenara — APPROVED** [review]
Code changes look fine. Please reply to all the comments as this will be needed for future reference.
Would be good if you expand on the PR description as well.

File: webpack.config.js (hunk 2 above)
Comment (Kesari3008) [reply webpack.config.js:21]: Sure, added

File: webpack.config.js (hunk 3 above)
Comment (Kesari3008) [reply webpack.config.js:80]: done

File: webpack.config.js (hunk 4 above)
Comment (Kesari3008) [reply webpack.config.js:99]: done

File: package.json (hunk 1 above)
Comment (Kesari3008) [reply package.json:36]: No reason, this change came on it's own somehow

**Pull request merged**